go.mod

```diff
@@ -2,11 +2,11 @@ module github.com/containerd/nri
 
 go 1.19
 
-// when updating containerd, adjust the replace rules accordingly
 require (
-	github.com/containerd/containerd v1.6.9
 	github.com/containerd/ttrpc v1.1.1-0.20220420014843-944ef4a40df3
 	github.com/moby/sys/mountinfo v0.6.2
+	github.com/onsi/ginkgo/v2 v2.5.0
+	github.com/onsi/gomega v1.24.0
 	github.com/opencontainers/runtime-spec v1.0.3-0.20220825212826-86290f6a00fb
 	github.com/opencontainers/runtime-tools v0.9.0
```

Comment:
Does anyone know if we still need the replace rule for this one?

```
replace github.com/opencontainers/runtime-tools v0.9.0 => github.com/opencontainers/runtime-tools v0.0.0-20221026201742-946c877fa809
```

Reply:
I think we do. v0.9.0 is still the tip of tags.

Reply:
But I suspect we could also replace the replace with a more straightforward import of `github.com/opencontainers/runtime-tools v0.9.1-0.20221107090550-2e043c6bd626`.